src/features/lightspeed/playbookGeneration.ts

```diff
@@ -184,7 +184,7 @@ export async function showPlaybookGenerationPage(
         }
 
         const html = await (
-          await require("../utils/syntaxHighlighter")
+          await require(/* webpackIgnore: true */ "../../../../syntaxHighlighter/src/syntaxHighlighter")
```

Comment:
`syntaxHighligher` is the code using the [shiki](https://shiki.style/) library, which is used in Playbook Generation feature. It is built with language/color theme definitions using webpack. To workaround an issue with Mocha, it is dynamically imported here. In order to tell webpack not to bundle syntaxHighligher in the client bundle (`extension.js`), a special comment is inserted here.